config/Simplenote.debug.xcconfig

```diff
@@ -1 +1,3 @@
-#include "Version.public.xcconfig"
+#include "Base.xcconfig"
+
+CODE_SIGN_IDENTITY = Apple Development
```

Comment:
```suggestion
CODE_SIGN_IDENTITY = Apple Development

// Notice we don't have a PROVISIONING_PROFILE_SPECIFIER defined here because this xcconfig is used at the project-level but that settings need to be target-level.
// Once we'll have target-level xcconfigs, we shall add the setting there
```